Time/shiftstats.30s.py

```diff
@@ -1,143 +0,0 @@
-#!/usr/bin/python
```

Comment:
What is broken or obsolete in this plug-in? Any chance of salvaging it?

Reply:
It depends on `/var/log/accountpolicy.log` which seems to unavailable for more than 6 years.